src/kbmod/run_search.py

```diff
@@ -382,7 +384,7 @@ def _count_known_matches(self, result_list, search):
             print(matches_string)
         print("-----------------")
 
-    def _calc_barycentric_corr(self, img_info, dist):
+    def _calc_barycentric_corr(self, wcslist, mjds, x_size, y_size, dist):
```

Comment:
Can it be named `wcs_list` so it matches previous

Reply:
Done